src/block/RespawnAnchor.php

```diff
@@ -68,9 +73,45 @@ public function onInteract(Item $item, int $face, Vector3 $clickVector, ?Player
 		}
 
 		if($this->charges > self::MIN_CHARGES){
-			//TODO: Implement the ability to set a respawn point in Nether
-			$this->explode($player);
-			return true;
+			if($player === null){
+				return false;
+			}
+
+			$ev = new PlayerRespawnAnchorUseEvent($player, $this, PlayerRespawnAnchorUseEvent::ACTION_EXPLODE);
+
+			if($this->position->getWorld()->getId() === DimensionIds::NETHER){
```

Comment:
I said _don't_ do this. This is never going to be the case.

The aim is to make plugins able to change the outcome.

Reply:
Honestly we probably ought to deprecate this function. It serves no obvious purpose. But that's a topic for another issue.